app/scripts/modules/core/src/pipeline/config/stages/overrideTimeout/OverrideTimeout.tsx

```diff
@@ -13,26 +14,32 @@ export interface IOverrideTimeoutConfigProps {
 }
 
 interface IStageConfig {
-  defaultTimeoutMs: number;
+  supportsCustomTimeout: boolean;
 }
 
+const toHoursAndMinutes = (ms: number) => {
+  if (!ms) {
+    return { hours: 0, minutes: 0 };
+  } else {
+    const seconds = ms / 1000;
```

Comment:
super minor nit: you could make it `minutes = ms / 60000` to make the subsequent lines slightly more readable.

Reply:
just noticed this was moved...

Reply:
eh, i was actually just moving this from down below in the file so intentionally didn't touch anything. since you mentioned i'll actually move it over to `luxon` calls which will (hopefully?) be even more straightforward

Reply:
i do these things because i care about you mark 35530d0